lib/rubocop/cop/style/symbol_proc.rb

```diff
@@ -100,6 +119,11 @@ def destructuring_block_argument?(argument_node)
 
         private
 
+        def contains_comment?(node)
```

Comment:
Why don't you use the `CommentsHelp` module instead? I think it provides exactly the same method there.

Reply:
I'd missed it. Thank you so much.